scrapers/SexBabesVR.yml

```diff
@@ -0,0 +1,29 @@
+name: "SexBabesVR"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - sexbabesvr.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="video-info"]
+    scene:
+      Title: $info//div[@class="video-group-left"]//h1[@class="title"]/text()
+      Date:
+        selector: $info//div[@class="video-group-left"]//div[@class="grid-one-date"]/span[@class="date-display-single"]/@content
```

Comment:
date seems to be like this `2016-09-05T22:00:00+00:00` for your selector
a replace is needed before parse date
eg
```yaml
        replace:
          - regex: (\d{4}-\d{2}-\d{2})T.+
            with: $1
```

Reply:
Thanks a bunch.  This works.  I've learned something new here as well. Fixed in code.